client/extensions/woocommerce/lib/order-status/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Lists of statuses in each group, waiting for payment, waiting for
+ * fulfillment, and finished orders.
+ */
+export const statusWaitingPayment = [
```

Comment:
Do you mean like the setting up constants for the `unpaid`/`unfulfilled`/`finished` statuses, or something else?